File: market/test/java/ru/yandex/market/partner/contract/config/FunctionalTestConfig.java
package ru.yandex.market.partner.contract.config;

import org.springframework.boot.test.context.TestConfiguration;
import org.springframework.context.annotation.Bean;

import ru.yandex.common.util.date.TestableClock;

/**
 * Основной конфиг для тестов
 */
@TestConfiguration
public class FunctionalTestConfig {

    @Bean
    public TestableClock clock() {
        return new TestableClock();
    }
}

File: market/test/kotlin/ru/yandex/market/integration/npd/FunctionalTestConfig.kt
package ru.yandex.market.integration.npd

import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration
import java.time.Clock
import java.time.OffsetDateTime
import java.time.ZoneId
import java.time.format.DateTimeFormatter

@Configuration
open class FunctionalTestConfig {

    @Bean
    open fun clock(): Clock {
        return Clock.fixed(
            OffsetDateTime.parse("2022-05-17T17:30:00+03:00", DateTimeFormatter.ISO_OFFSET_DATE_TIME).toInstant(),
            ZoneId.of("Europe/Moscow")
        )
    }
}
